### ISSUE TITLE
routeEnhancers from the documention returns parsing error

### ISSUE THREAD

**Issue by fischhase** [georgringer/news#1291]
Hallo Georg,

I try to setup a routeEnhancer for news and used your easy example from the news documentation. Anyway I always get an error:

> (1/1) Symfony\Component\Yaml\Exception\ParseException
> Unable to parse at line 29 (near " _controller: 'News::detail'").

I also tried many other examples from the web. I got always the same error message. I use TYPO3 9.5.18 and news 8.3.0 and News events 4.0.0. I can't explain why 'News::detail' ist a problem;  and its not even 'detail', but also 'list'. 

Do I have to setup some additional parameters (TCEMAIN, SETUP, Pluginsettings, LocalConfiguration ...) 
Here ist my tiny config.yml
[config.yaml.txt](https://github.com/georgringer/news/files/4723199/config.yaml.txt)

Thank you
Martin

**Comment by JarvisH** [comment]
It seems your yaml indentations may be wrong. Please compare with this working example:
```yaml
-
    routePath: '/{news-title}'
    _controller: 'News::detail'
    _arguments:
    news-title: news
```

**Comment by georgringer** [comment]
closing as it is about the wrong intentation

**Comment by fischhase** [comment]
Hallo Georg,

thank you for your answer. I did not get a mail on friday, otherwise I would have answered earlier. 
Unfortunately that snipped you sent was not the solution. 

`routes:
            -
                routePath: '/{news-title}'
                _controller: 'News::detail'
                _arguments:
                news-title: news
`
generates no error but also does not change the url :-/

In another project with TYPO3 10 the example works out of the box. I tried to use the config.yaml of this working project (of cause with the necessary changes). Now I get the Error Message:

> (1/1) Symfony\Component\Routing\Exception\InvalidParameterException
> Parameter "tx_news_pi1__news" for route "tx_news_pi1_0" must match "[^/]++" ("" given) to generate a corresponding URL.

There seams to be something empty that should no be empty... I attached error log and text.yaml 
[config.txt](https://github.com/georgringer/news/files/4752033/config.txt)
[TYPO3 Exception.pdf](https://github.com/georgringer/news/files/4752034/TYPO3.Exception.pdf)
Thank you.
Martin
